### PR TITLE
chore: remove align prop from Text & Heading for react-native

### PR DESCRIPTION
**Pull request by deorukhkarchaitanya**

### CODE REVIEW

**Comment by deorukhkarchaitanya** [comment]
@shubhnik @shridhar52war please checkout to this PR and see if your components are affected. I had to update snapshots for all your components. I've done a visual check and things seem okay. Please approve this once you've gone through the components.

**Comment by dhruvdutt** [comment]
Why are we removing this prop?

**Comment by kamleshchandnani** [comment]
> Why are we removing this prop?
Alignment should be handled by parent and not the component itself. Rendering the text component and doing text alignment won't make any difference. We have enhancers for this purpose

File: src/atoms/Text/Text.native.js [hunk 1]
```diff
@@ -50,14 +39,12 @@ const StyledText = styled(NativeText)`
   font-size: ${styles.fontSize};
   color: ${styles.color};
   text-decoration-line: ${styles.textDecorationLine};
```
Comment (kamleshchandnani) [inline src/atoms/Text/Text.native.js]: you didn't had to mention align left as a defualt style as you were mentioning the other day? It takes it by default?

Comment (deorukhkarchaitanya) [reply src/atoms/Text/Text.native.js]: The parent will take care

**Pull request merged**